categories/software-development/rules-to-better-openapi-specification.md

```diff
@@ -1,14 +1,9 @@
 ---
 type: category
-title: Rules to Better OpenAPI Specification
+title: Rules to Better API Management
 guid: 680c1175-a54f-451a-ace8-9b0b988bb783
 uri: rules-to-better-openapi-specification
 index:
-- the-best-openapi-toolchain
-- give-developers-an-api-help-page
-- automate-the-generation-of-your-api-specification-and-required-clients
-- define-your-response-types
-
----
```

Comment:
I think you accidentally removed the `---` from underneath the frontmatter. Please add it back